net/ampr-ripd/files/ampr-ripd-init

```diff
@@ -17,14 +17,21 @@ start() {
 
 		exit 1
 	fi
+	config_load ampr-ripd
+	config_get enabled network enabled 0
```

Comment:
I find this a bit confusing and inconsistent that this defaults to 0, while the config option and `configure` - to 1.

Reply:
With regard to running configure -- if the default config is present then the software is not configured for use.  No one is expected to actually use the provided defaults, they are documentation placeholders.  So yes, running configure is necessary.

Therefore, the value of enabled in the distributed default config does not matter.  If the default config is detected, the user is going to have to run configure to provide their assigned network information.

Reply:
I still see:

```
	option enabled '1'
```

in the config from this PR.